server/src/main/scala/org/http4s/server/middleware/Logger.scala

```diff
@@ -2,30 +2,35 @@ package org.http4s
 package server
 package middleware
 
+import cats.data.Kleisli
 import cats.effect._
 import fs2._
 import org.http4s.util.CaseInsensitiveString
+import org.log4s.getLogger
+import scala.concurrent.ExecutionContext
 
 /**
   * Simple Middleware for Logging All Requests and Responses
   */
 object Logger {
+  private[this] val logger = getLogger
+
   def apply[F[_]: Effect](
       logHeaders: Boolean,
       logBody: Boolean,
-      redactHeadersWhen: CaseInsensitiveString => Boolean = Headers.SensitiveHeaders.contains
-  )(@deprecatedName('httpService) httpRoutes: HttpRoutes[F]): HttpRoutes[F] =
-    ResponseLogger(logHeaders, logBody, redactHeadersWhen)(
-      RequestLogger(logHeaders, logBody, redactHeadersWhen)(
-        httpRoutes
-      )
+      redactHeadersWhen: CaseInsensitiveString => Boolean = Headers.SensitiveHeaders.contains,
+      logAction: String => Unit = logger.info(_)
+  )(@deprecatedName('httpService) http: Kleisli[F, Request[F], Response[F]])(
```

Comment:
You could use the `HttpApp` alias here, or alternatively, turn this into the parametric `F ~> G` style we have in a bunch of places now.